### PR TITLE
API connection base code and reformat of code

### PR DESCRIPTION
**Pull request by Nipa-Code**
Create API connection with aiohttp, reformat code with black and help of flake8. Create tox.ini for ignoring some flake8 errors/warnings. I placed the most stuff in commit message :/
Saying it again: Code is NOT tested.

### CODE REVIEW

**Review by sifte — APPROVED** [review]

**Comment by Yat-o** [comment]
so what you're trying to say is you made this big ass pr without testing any of the code….

File: src/bot.py [hunk 1]
```diff
@@ -39,13 +40,19 @@ def __init__(self):
             strip_after_prefix=True,
             help_command=None,  # type: ignore
             intents=Intents.all(),
-            allowed_mentions=AllowedMentions(everyone=False, roles=False)
+            allowed_mentions=AllowedMentions(everyone=False, roles=False),
```
Comment (Pandabweer) [inline src/bot.py:43]: Unnecessary extra comma.

File: src/bot.py [hunk 2]
```diff
@@ -10,6 +10,7 @@
 
 from src.utils.utils import main_embed
 from .HIDDEN import TOKEN, MONGO_URI
+import aiohttp
```
Comment (Pandabweer) [inline src/bot.py:13]: For continuation and consistency import are placed a the top, from import below.

File: src/cogs/config.py [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
+    Cog,
+    BucketType,
     CooldownMapping,
-    Bot, 
-    Context, 
+    Bot,
+    Context,
     command,
-    has_permissions
+    has_permissions,
```
Comment (Pandabweer) [inline src/cogs/config.py:10]: Unnecessary extra comma again.

File: src/cogs/errors.py [hunk 4]
```diff
@@ -101,7 +101,7 @@ async def on_command_error(self, ctx: Context, error: Exception):
             await ctx.send(
                 embed=Embed(
                     description=f"I am missing `{self.perms_parser(error.missing_permissions)}` permissions required to run the command",
-                    color=Color.red()
+                    color=Color.red(),
```
Comment (Pandabweer) [inline src/cogs/errors.py:104]: Unnecessary extra comma again.

File: src/cogs/config.py [hunk 5]
```diff
@@ -22,50 +22,49 @@ class Config(
...
-        'cconfig',
-        'createconf',
-    ])
+    @command(
+        aliases=[
+            "createconfig",
+            "newconfig",
+            "makeconfig",
+            "cconfig",
+            "createconf",
```
Comment (Pandabweer) [inline src/cogs/config.py:58]: Unnecessary extra comma again.

File: src/cogs/config.py [hunk 6]
```diff
@@ -22,50 +22,49 @@ class Config(
...
+        ]
+    )
     @has_permissions(
         manage_guild=True,
```
Comment (Pandabweer) [inline src/cogs/config.py:62]: Unnecessary extra comma again (i know some aren't your fault).

File: src/cogs/help.py [hunk 7]
```diff
@@ -98,7 +98,13 @@ async def send_bot_help(self, mapping: Mapping[Optional[Cog], List[Command]]):
             url="https://media.discordapp.net/attachments/926115595307614252/927951464725377034/big.png"
         )
         for cog_name in _bot.cogs:
-            if cog_name.lower() in ("jishaku", "helpcog", "staff", "errorhandler", "ide"):
+            if cog_name.lower() in (
+                "jishaku",
+                "helpcog",
+                "staff",
+                "errorhandler",
+                "ide",
```
Comment (Pandabweer) [inline src/cogs/help.py:106]: Unnecessary extra comma again.

File: src/cogs/help.py [hunk 8]
```diff
@@ -129,10 +135,13 @@ async def send_command_help(self, cmd: Command):
         """
         command_help_dict = {
```
Comment (Pandabweer) [inline src/cogs/help.py:136]: This is also not your fault but when having a lot of 'or' operators please use the `any()` function that Python has.

File: src/cogs/help.py [hunk 9]
```diff
@@ -195,7 +204,13 @@ def __init__(self, ctx: Context) -> None:
         self.context: Context = ctx
         options = []
         for cog_name in ctx.bot.cogs:
-            if cog_name.lower() in ("jishaku", "helpcog", "staff", "ide", "errorhandler"):
+            if cog_name.lower() in (
+                "jishaku",
+                "helpcog",
+                "staff",
+                "ide",
+                "errorhandler",
```
Comment (Pandabweer) [inline src/cogs/help.py:212]: Unnecessary extra comma again.

File: src/cogs/help.py [hunk 10]
```diff
@@ -212,8 +227,8 @@ def __init__(self, ctx: Context) -> None:
     async def callback(self, interaction: MessageInteraction):
         if interaction.author.id != self.context.author.id:
             return await interaction.response.send_message(
-                f'This command was invoked by {self.context.author}\nOnly they can use it',
-                ephemeral=True
+                f"This command was invoked by {self.context.author}\nOnly they can use it",
+                ephemeral=True,
```
Comment (Pandabweer) [inline src/cogs/help.py:231]: Unnecessary extra comma again.

File: src/cogs/ide/auto_detect.py [hunk 11]
```diff
@@ -22,17 +22,17 @@
     "sinh": lambda x: math.sinh(x),
     "cosh": lambda x: math.cosh(x),
     "tanh": lambda x: math.tanh(x),
-    "abs": lambda x: math.fabs(x)
+    "abs": lambda x: math.fabs(x),
```
Comment (Pandabweer) [inline src/cogs/ide/auto_detect.py:25]: Unnecessary extra comma again.

File: src/cogs/ide/dialogs/edit_view.py [hunk 12]
```diff
@@ -40,17 +40,20 @@ def page_integrity(page: int, pages: int, method: Literal["back", "next"]):
...
+        )
         self.parent = parent
         self.bot_message = self.parent.parent.bot_message
         self.ctx = self.parent.parent.ctx
 
     async def callback(self, interaction: disnake.MessageInteraction):
         await interaction.response.send_message(
-            f"Successfully restored version!",
-            ephemeral=True
+            "Successfully restored version!", ephemeral=True
```
Comment (Pandabweer) [inline src/cogs/ide/dialogs/edit_view.py:52]: Try to stay consistent, at other files you placed ephemeral on a separate line and now you kept it here.

File: src/cogs/ide/dialogs/edit_view.py [hunk 13]
```diff
@@ -109,7 +118,7 @@ def __init__(self, parent: EditView):
         self.options = [
             disnake.SelectOption(value="1", label="Find"),
             disnake.SelectOption(value="2", label="Go to page..."),
-            disnake.SelectOption(value="3", label="Version History")
+            disnake.SelectOption(value="3", label="Version History"),
```
Comment (Pandabweer) [inline src/cogs/ide/dialogs/edit_view.py:121]: Unnecessary extra comma again.

File: src/cogs/ide/dialogs/edit_view.py [hunk 14]
```diff
@@ -125,25 +134,32 @@ async def find_option(self, interaction: disnake.MessageInteraction):
             "For advanced use look the available flags below:\n"
             "\t`-replace <*chars>`: Replace every occurrence with `chars`\n"
             "\n**Example:**\n```\n-replace foo\nbar\n```",
-            ephemeral=True
+            ephemeral=True,
```
Comment (Pandabweer) [inline src/cogs/ide/dialogs/edit_view.py:137]: Unnecessary extra comma again.

File: src/cogs/ide/dialogs/edit_view.py [hunk 15]
```diff
@@ -125,25 +134,32 @@ async def find_option(self, interaction: disnake.MessageInteraction):
...
         )
         content: str = (
             await self.ctx.bot.wait_for(
                 "message",
-                check=lambda m: m.author == interaction.author and m.channel == interaction.channel,
+                check=lambda m: m.author == interaction.author
+                and m.channel == interaction.channel,
```
Comment (Pandabweer) [inline src/cogs/ide/dialogs/edit_view.py:143]: Unnecessary extra comma again.

File: src/cogs/ide/dialogs/edit_view.py [hunk 16]
```diff
@@ -166,7 +182,7 @@ async def find_option(self, interaction: disnake.MessageInteraction):
                 check=lambda m: m.author == interaction.author
                 and m.channel == interaction.channel
                 and m.content.lower() in ("back", "next", "quit"),
-                timeout=60
+                timeout=60,
```
Comment (Pandabweer) [inline src/cogs/ide/dialogs/edit_view.py:185]: Unnecessary extra comma again.

File: src/cogs/ide/dialogs/edit_view.py [hunk 17]
```diff
@@ -193,7 +209,8 @@ async def goto_option(self, interaction: disnake.MessageInteraction):
         await interaction.response.send_message("Enter page number...", ephemeral=True)
         message: disnake.Message = await self.ctx.bot.wait_for(
             "message",
-            check=lambda m: m.author == interaction.author and m.channel == interaction.channel,
+            check=lambda m: m.author == interaction.author
+            and m.channel == interaction.channel,
```
Comment (Pandabweer) [inline src/cogs/ide/dialogs/edit_view.py:213]: Unnecessary extra comma again.

File: src/cogs/ide/dialogs/edit_view.py [hunk 18]
```diff
@@ -284,7 +307,7 @@ def create_undo(self):
     @property
     def pages(self):
         lines = add_lines(self.file.content)
-        return ["".join(lines[x: x + 50]) for x in range(0, len(lines), 50)]
+        return ["".join(lines[x : x + 50]) for x in range(0, len(lines), 50)]
```
Comment (Pandabweer) [inline src/cogs/ide/dialogs/edit_view.py:310]: PEP8 states index or dict key-value assignments should be `[x: x + 50]` this should be reversed.

File: src/cogs/ide/dialogs/edit_view.py [hunk 19]
```diff
@@ -324,7 +347,8 @@ async def replace_button(
         content: str = (
             await self.ctx.bot.wait_for(
                 "message",
-                check=lambda m: m.author == interaction.author and m.channel == interaction.channel,
+                check=lambda m: m.author == interaction.author
+                and m.channel == interaction.channel,
```
Comment (Pandabweer) [inline src/cogs/ide/dialogs/edit_view.py:351]: Unnecessary extra comma again.

File: src/cogs/ide/dialogs/edit_view.py [hunk 20]
```diff
@@ -341,15 +365,15 @@ async def replace_button(
             from_, to = 0, len(self.file.content) - 1
             code = clear_codeblock(content)
         sliced = self.file.content.splitlines()
-        del sliced[from_: to + 1]
+        del sliced[from_ : to + 1]
```
Comment (Pandabweer) [inline src/cogs/ide/dialogs/edit_view.py:368]: Again this should be `sliced[from_: to + 1]`

File: src/cogs/ide/dialogs/edit_view.py [hunk 21]
```diff
@@ -359,32 +383,30 @@ async def append_button(
             (
                 await self.ctx.bot.wait_for(
                     "message",
-                    check=lambda m: m.author == interaction.author and m.channel == interaction.channel,
+                    check=lambda m: m.author == interaction.author
+                    and m.channel == interaction.channel,
```
Comment (Pandabweer) [inline src/cogs/ide/dialogs/edit_view.py:387]: Unnecessary extra comma again.

File: src/cogs/ide/dialogs/edit_view.py [hunk 22]
```diff
@@ -359,32 +383,30 @@ async def append_button(
...
-            self, button: disnake.ui.Button, interaction: disnake.MessageInteraction
+        self, button: disnake.ui.Button, interaction: disnake.MessageInteraction
     ):
         await interaction.response.send_message(
             "Hello! Welcome to writer mode. In this mode every message you send will write to your open file\n"
             "If you want to the bot to ignore your message, start it with !\n"
             "To end this mode either click another button or type q\n"
             "This will time out after 5 minutes without a response\n",
-            ephemeral=True
+            ephemeral=True,
```
Comment (Pandabweer) [inline src/cogs/ide/dialogs/edit_view.py:402]: Unnecessary extra comma again.

File: src/cogs/ide/dialogs/edit_view.py [hunk 23]
```diff
@@ -359,32 +383,30 @@ async def append_button(
...
-            message:=
-            await self.ctx.bot.wait_for(
-                "message",
-                check=lambda m: m.author == interaction.author
-                and m.channel == interaction.channel,
-            )
+        while message := await self.ctx.bot.wait_for(
+            "message",
+            check=lambda m: m.author == interaction.author
+            and m.channel == interaction.channel,
```
Comment (Pandabweer) [inline src/cogs/ide/dialogs/edit_view.py:409]: Unnecessary extra comma again.

File: src/cogs/ide/dialogs/edit_view.py [hunk 24]
```diff
@@ -468,14 +494,15 @@ async def redo_button(
...
+        self, button: disnake.ui.Button, interaction: disnake.MessageInteraction
     ):
         await interaction.response.send_message(
             "What would you like the filename to be?", ephemeral=True
         )
         filename = await self.bot.wait_for(
             "message",
-            check=lambda m: self.ctx.author == m.author and m.channel == self.ctx.channel,
+            check=lambda m: self.ctx.author == m.author
+            and m.channel == self.ctx.channel,
```
Comment (Pandabweer) [inline src/cogs/ide/dialogs/edit_view.py:505]: Unnecessary extra comma again.

File: src/cogs/ide/dialogs/file_view.py [hunk 25]
```diff
@@ -95,15 +97,15 @@ async def first_button(
...
     ):
         content = self.file.content
         name = self.extension
 
         async with aiohttp.ClientSession() as session:
             async with session.post(
-                    url="https://emkc.org/api/v1/piston/execute",
-                    json={"language": name, "source": content},
+                url="https://emkc.org/api/v1/piston/execute",
+                json={"language": name, "source": content},
```
Comment (Pandabweer) [inline src/cogs/ide/dialogs/file_view.py:108]: Unnecessary extra comma again.

File: src/cogs/ide/dialogs/file_view.py [hunk 26]
```diff
@@ -156,28 +159,29 @@ async def third_button(
                 "".join(content[:50]),
                 self.file.filename,
                 self.file.extension,
-                f"\n1/{math.ceil(len(content) / 50)}"
+                f"\n1/{math.ceil(len(content) / 50)}",
```
Comment (Pandabweer) [inline src/cogs/ide/dialogs/file_view.py:162]: Unnecessary extra comma again.

File: src/cogs/ide/dialogs/file_view.py [hunk 27]
```diff
@@ -156,28 +159,29 @@ async def third_button(
...
+        self, button: disnake.ui.Button, interaction: disnake.MessageInteraction
     ):
         await interaction.response.send_message(
             "What would you like the filename to be?", ephemeral=True
         )
         filename = await self.bot.wait_for(
             "message",
-            check=lambda m: self.ctx.author == m.author and m.channel == self.ctx.channel,
+            check=lambda m: self.ctx.author == m.author
+            and m.channel == self.ctx.channel,
```
Comment (Pandabweer) [inline src/cogs/ide/dialogs/file_view.py:184]: Unnecessary extra comma again.

File: src/cogs/ide/dialogs/file_view.py [hunk 28]
```diff
@@ -197,18 +201,19 @@ async def rename_button(
...
         num = 0
         while not (
-                channel := await self.bot.wait_for(
-                    "message",
-                    check=lambda m: self.ctx.author == m.author and m.channel == self.ctx.channel,
-                )
+            channel := await self.bot.wait_for(
+                "message",
+                check=lambda m: self.ctx.author == m.author
+                and m.channel == self.ctx.channel,
```
Comment (Pandabweer) [inline src/cogs/ide/dialogs/file_view.py:215]: Unnecessary extra comma again.

File: src/cogs/ide/dialogs/open_view.py [hunk 29]
```diff
@@ -119,7 +119,7 @@ async def github_button(
                 "message",
                 check=lambda m: self.ctx.author == m.author
                 and m.channel == self.ctx.channel
-                and not self.is_exited
+                and not self.is_exited,
```
Comment (Pandabweer) [inline src/cogs/ide/dialogs/open_view.py:122]: Unnecessary extra comma again.

File: src/cogs/ide/dialogs/open_view.py [hunk 30]
```diff
@@ -60,7 +60,7 @@ async def upload_button(
...
```
Comment (Pandabweer) [inline src/cogs/ide/dialogs/open_view.py:63]: Unnecessary extra comma again.

File: src/cogs/ide/dialogs/open_view.py [hunk 31]
```diff
@@ -150,7 +150,8 @@ async def github_button(
                 content = (await b.text()).replace("`", "`​")
                 if content == "404: Not Found":
                     await interaction.send(
-                        "Invalid github link, please exit the IDE and try again.", delete_after=5
+                        "Invalid github link, please exit the IDE and try again.",
+                        delete_after=5,
```
Comment (Pandabweer) [inline src/cogs/ide/dialogs/open_view.py:154]: Unnecessary extra comma again.

File: src/cogs/ide/dialogs/open_view.py [hunk 32]
```diff
@@ -160,9 +161,9 @@ async def github_button(
 
                 content = base64.b64decode(content).decode("utf-8")
         if start and end:
-            content = "\n".join(content.splitlines()[int(start)-1:int(end)])
+            content = "\n".join(content.splitlines()[int(start) - 1 : int(end)])
```
Comment (Pandabweer) [inline src/cogs/ide/dialogs/open_view.py:164]: Line should be:
`content = "\n".join(content.splitlines()[int(start) - 1: int(end)])`

File: src/cogs/ide/dialogs/open_view.py [hunk 33]
```diff
@@ -196,7 +197,7 @@ async def link_button(
                 "message",
                 check=lambda m: self.ctx.author == m.author
                 and m.channel == self.ctx.channel
-                and not self.is_exited
+                and not self.is_exited,
```
Comment (Pandabweer) [inline src/cogs/ide/dialogs/open_view.py:200]: Unnecessary extra comma again.

File: src/cogs/ide/dialogs/open_view.py [hunk 34]
```diff
@@ -220,7 +221,7 @@ async def link_button(
             "message",
             check=lambda m: self.ctx.author == m.author
             and m.channel == self.ctx.channel
-            and not self.is_exited
+            and not self.is_exited,
```
Comment (Pandabweer) [inline src/cogs/ide/dialogs/open_view.py:224]: Unnecessary extra comma again.

File: src/cogs/ide/dialogs/open_view.py [hunk 35]
```diff
@@ -254,7 +255,7 @@ async def create_button(
...
```
Comment (Pandabweer) [inline src/cogs/ide/dialogs/open_view.py:258]: Unnecessary extra comma again.

File: src/cogs/ide/dialogs/open_view.py [hunk 36]
```diff
@@ -294,7 +295,7 @@ async def saved_button(
 
         embed = EmbedFactory.ide_embed(
             self.ctx,
-            f"Open a file from saved:\nCurrent directory: /",
+            "Open a file from saved:\nCurrent directory: /",
```
Comment (Pandabweer) [inline src/cogs/ide/dialogs/open_view.py:298]: Unnecessary extra comma again.

File: src/cogs/ide/dialogs/open_view.py [hunk 37]
```diff
@@ -318,21 +319,34 @@ async def exit_button(
...
+    ):
         responses = []
 
-        for iteration, question in enumerate(['sum up your report in less than 10 words', 'explain your report. present as detailed of a description as you can provide, including button clicks, errors shown (if any), file open, and intention'], start=1):
+        for iteration, question in enumerate(
+            [
+                "sum up your report in less than 10 words",
+                "explain your report. present as detailed of a description as you can provide, including button clicks, errors shown (if any), file open, and intention",
+            ],
+            start=1,
```
Comment (Pandabweer) [inline src/cogs/ide/dialogs/open_view.py:330]: Unnecessary extra comma again.

File: src/cogs/ide/dialogs/open_view.py [hunk 38]
```diff
@@ -318,21 +319,34 @@ async def exit_button(
...
+        ):
             if iteration == 1:
-                await interaction.response.send_message(f"Please {question}\nType q to end your report\nQuestion number {iteration}/2", ephemeral=True)
+                await interaction.response.send_message(
+                    f"Please {question}\nType q to end your report\nQuestion number {iteration}/2",
+                    ephemeral=True,
```
Comment (Pandabweer) [inline src/cogs/ide/dialogs/open_view.py:335]: Unnecessary extra comma again.

File: src/cogs/ide/dialogs/open_view.py [hunk 39]
```diff
@@ -341,10 +355,10 @@ async def report_button(
         embed = disnake.Embed(
             title=responses[0],
             description="```yaml\n" + responses[0] + "```",
-            timestamp=interaction.message.created_at
+            timestamp=interaction.message.created_at,
```
Comment (Pandabweer) [inline src/cogs/ide/dialogs/open_view.py:358]: Unnecessary extra comma again.

File: src/cogs/ide/dialogs/open_view.py [hunk 40]
```diff
@@ -341,10 +355,10 @@ async def report_button(
...
         ).set_author(
             name=f"From {interaction.author.name}",
-            icon_url=interaction.author.avatar.url
+            icon_url=interaction.author.avatar.url,
```
Comment (Pandabweer) [inline src/cogs/ide/dialogs/open_view.py:361]: Unnecessary extra comma again.

File: src/cogs/ide/ide.py [hunk 41]
```diff
@@ -38,7 +38,7 @@ async def check_activity(self):
 
     @commands.command(
         help="""Have you used the linux commandline editor, nano? This discord text editor is like nano , and implements safe, reliable and fast file storing with editing and compiling technology. The database is secure and cannot be accessed or broken into by anyone, not even the core developers. You can upload or create files and these files would be saved into a filesystem which you can open at any time. If you have an open file you can compile it and run it (depending on the filetype). You can also edit the content and replace text. You can also pull and push to github depending on the file/folder you uploaded.""",
-        aliases = ['texteditor', 'editor']
+        aliases=["texteditor", "editor"],
```
Comment (Pandabweer) [inline src/cogs/ide/ide.py:41]: Unnecessary extra comma again.

File: src/cogs/misc.py [hunk 42]
```diff
@@ -127,31 +127,37 @@ async def jail(self, ctx: Context, *, member: Member = None):
...
+    @command(aliases=["bug", "broken"])
     async def report(self, ctx: Context):
         responses = []
 
-        for iteration, question in enumerate(['sum up your report in less than 10 words', 'explain your report. present as detailed of a description as you can provide, including button clicks, errors shown (if any), file open, and intention'], start=1):
-            await ctx.send(f"Please {question}\nType q to end your report\nQuestion number {iteration}/2")
+        for iteration, question in enumerate(
+            [
+                "sum up your report in less than 10 words",
+                "explain your report. present as detailed of a description as you can provide, including button clicks, errors shown (if any), file open, and intention",
```
Comment (Pandabweer) [inline src/cogs/misc.py:138]: This line is still way too long, thanks for shortening it just a bit.

File: src/cogs/misc.py [hunk 43]
```diff
@@ -127,31 +127,37 @@ async def jail(self, ctx: Context, *, member: Member = None):
...
 
         embed = Embed(
             title=responses[0],
             description="```yaml\n" + responses[0] + "```",
-            timestamp=ctx.message.created_at
-        ).set_author(
-            name=f"From {ctx.author.name}",
-            icon_url=ctx.author.avatar.url
-        )
+            timestamp=ctx.message.created_at,
```
Comment (Pandabweer) [inline src/cogs/misc.py:159]: Unnecessary extra comma again.

File: src/utils/paginator.py [hunk 44]
```diff
@@ -252,12 +252,12 @@ def get_pages(self):
...
         while True:
             if len(text) != 0:
-                new_text = text[0: self.breakpoint]
+                new_text = text[0 : self.breakpoint]
                 if self.prefix:
                     new_text = self.prefix + "\n" + new_text
                 if self.suffix:
                     new_text = new_text + "\n" + self.suffix
-                text = text[self.breakpoint:]
+                text = text[self.breakpoint :]
```
Comment (Pandabweer) [inline src/utils/paginator.py:260]: : is smashed against it. Reverse.

File: src/cogs/mod.py [hunk 45]
```diff
@@ -165,14 +180,21 @@ async def unmute(self, ctx, member: disnake.Member, *, reason=None):
...
     @commands.bot_has_permissions(manage_roles=True)
-    async def role(self, ctx, member: disnake.Member, role: disnake.Role, *, reason="No Reason Provided."):
-        """ Add a role to the user"""
+    async def role(
+        self,
+        ctx,
+        member: disnake.Member,
+        role: disnake.Role,
+        *,
+        reason="No Reason Provided.",
```
Comment (Pandabweer) [inline src/cogs/mod.py:189]: Unnecessary extra comma again.

File: src/cogs/mod.py [hunk 46]
```diff
@@ -110,7 +120,12 @@ async def unban(
     @commands.guild_only()
     @commands.has_permissions(manage_channels=True)
     @commands.bot_has_permissions(manage_channels=True)
-    async def slowmode(self, ctx: Context, channel: Greedy[disnake.TextChannel] = None, slowmode: int = None):
+    async def slowmode(
+        self,
+        ctx: Context,
+        channel: Greedy[disnake.TextChannel] = None,
+        slowmode: int = None,
```
Comment (Pandabweer) [inline src/cogs/mod.py:127]: Unnecessary extra comma again.

Comment (Pandabweer) [inline src/cogs/mod.py:127]: Unnecessary extra comma again.

File: src/cogs/mod.py [hunk 47]
```diff
@@ -45,27 +48,32 @@ async def kick(
...
     @commands.bot_has_permissions(ban_members=True)
     async def ban(
-            self,
-            ctx: Context,
-            member: disnake.Member,
-            reason="No Reason Provided.",
+        self,
+        ctx: Context,
+        member: disnake.Member,
+        reason="No Reason Provided.",
```
Comment (Pandabweer) [inline src/cogs/mod.py:65]: Unnecessary extra comma again.

File: src/cogs/mod.py [hunk 48]
```diff
@@ -23,18 +23,21 @@ def __init__(self, bot: commands.Bot) -> None:
...
     @commands.bot_has_permissions(kick_members=True)
     async def kick(
-            self,
-            ctx: Context,
-            member: disnake.Member,
-            reason: str = "No Reason Provided.",
+        self,
+        ctx: Context,
+        member: disnake.Member,
+        reason: str = "No Reason Provided.",
```
Comment (Pandabweer) [inline src/cogs/mod.py:29]: Unnecessary extra comma again.

**Review by Pandabweer — CHANGES_REQUESTED** [review]
Consistency is a bit wonky and there are some small fixes but overall great cleanup of single quotes, unnecessary f-strings and a lot of long code shortened.

File: src/cogs/help.py (hunk 8 above)
Comment (NotFlameDev) [reply src/cogs/help.py:136]: That's not a conditional statement, so you can't use `any()` there

**Review by NotFlameDev — COMMENTED** [review]
Removing everything in `__init__.py` raises a bunch of `ImportError`

**Comment by Nipa-Code** [comment]
@NotFlameDev they were unused imports tho

**Comment by NotFlameDev** [comment]
> @NotFlameDev they were unused imports tho

they are unused in `__init__.py` but it's used in other files. And without them, you can't import the stuff

**Review by NotFlameDev — APPROVED** [review]

File: src/bot.py [hunk 49]
```diff
@@ -47,12 +46,7 @@ def __init__(self):
         self.error_channel = None
         self.server_message = None
         self.bugs = range(10000, 100000)
-        self.session: aiohttp.ClientSession = self.loop.run_until_complete(
-            self.create_session()
-        )
-
-    async def create_session(self) -> aiohttp.ClientSession:
-        return aiohttp.ClientSession()
+        self.session = None
```
Comment (Pandabweer) [inline src/bot.py:49]: Define a session

**Review by Pandabweer — APPROVED** [review]

**Pull request merged**